src/style.css

```diff
@@ -1,3 +1,64 @@
+:root {
+  --blue: #00acdc;
+}
+
 body {
+  font-family: Roboto, Helvetica, Arial, sans-serif;
+  margin: 0;
+  background-color: aqua;
+  color: var(--blue);
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  background-color: white;
+}
+
+.logo {
```

Comment:
Be consistent with your *vertical* margins (Add only top or only bottom margin, but not both). Vertical margins can be collapsed in some cases.